src/api/urls.py

```diff
@@ -22,67 +23,60 @@
 from .v3.single_page import single_page
 
 urlpatterns = [
-    url(r"regions/$", regions),
-    url(r"regions/live/$", liveregions),
-    url(r"regions/hidden/$", hiddenregions),
-    url(r"regions/pushnew/$", pushnew),
+    url(r"regions/?$", regions),
+    url(r"regions/live/?$", liveregions),
+    url(r"regions/hidden/?$", hiddenregions),
+    url(r"regions/pushnew/?$", pushnew),
     url(
         r"(?P<region_slug>[-\w]+)/",
         include(
             [
-                url(r"languages$", languages),
-                url(r"offers$", offers),
+                url(r"languages/?$", languages),
+                url(r"offers/?$", offers),
```

Comment:
```suggestion
                url(r"(extras|offers)/?$", offers),
```
Should we add both urls here as well?

Reply:
yes